R/run_example.R

```diff
@@ -121,13 +134,32 @@ run_example <- function(){
 
         #####################Create third panel################################
         shiny::tabPanel(
-          "Feedback Types",
+          "Tables and tabs",
           value = "panel3",
           gov_layout(
             size = "two-thirds",
-            backlink_Input("back2"),
+            backlink_Input("back1"),
             heading_text("Page 3", size = "l"),
-            label_hint("label3", "These are some examples of the types of user
+            label_hint("label3", "These are some examples of using tabs and
+                       tables"),
+            heading_text("govTable", size = "s"),
+            shinyGovstyle::govTable(
```

Comment:
The table title is coming out in grey - don't know if it's for this PR or another one, but should be a quick CSS change to make it black? 

GDS component
![image](https://github.com/user-attachments/assets/7a580d53-ed8f-47b7-a402-049647c5fc28)

`run_example()` on this branch
![image](https://github.com/user-attachments/assets/ce811ea9-da22-4f7f-822b-37e6d6a70c22)

Reply:
Fixed in the css